packages/datasets/src/EnvVars.ts

```diff
@@ -7,6 +7,12 @@ export const OPENAI_ENV_VARS = {
 
 export const HUGGINGFACE = {
   HUGGINGFACE_ACCESS_TOKEN: "",
+};
+
+export const HUGGINGFACE_DOCS_CONTENT = {
   HUGGINGFACE_DOCS_CONTENT_REPO: "",
   HUGGINGFACE_DOCS_CODE_EXAMPLE_REPO: "",
 };
```

Comment:
```suggestion
export const HUGGINGFACE_DOCS_CONTENT = {
  HUGGINGFACE_DOCS_CONTENT_REPO: "",
};
```

Should remove this since it's now in `HUGGINGFACE_DOCS_CODE_EXAMPLES`?

Reply:
i think you're right..will check